generic/tests/linux_stress.py

```diff
@@ -19,19 +22,41 @@ def run(test, params, env):
     :param env: Dictionary with test environment
     """
 
-    vm = env.get_vm(params['main_vm'])
-    vm.verify_alive()
-    stress = utils_test.VMStress(vm, 'stress', params)
-    stress.load_stress_tool()
-    stress_duration = int(params.get('stress_duration', 0))
-    # NOTE: stress_duration = 0 ONLY for some legacy test cases using
-    # autotest stress.control as their sub test.
-    # Please DO define stress_duration to make sure the clean action
-    # being performed, if your case can not be controlled by time,
-    # use utils_test.VMStress() directly
+    stress_duration = int(params.get("stress_duration", ""))
+    vms = env.get_all_vms()
+    up_time = {}
+    error = 0
```

Comment:
let's denote it as `False` and flip it to `True` on failure.

Reply:
Taken care.